ui/helpers/utils/token-util.js

```diff
@@ -12,10 +12,16 @@ import { formatCurrency } from './confirm-tx.util';
 const DEFAULT_SYMBOL = '';
 
 async function getSymbolFromContract(tokenAddress) {
-  const token = util.getContractAtAddress(tokenAddress);
+  const contract = util.getContractAtAddress(tokenAddress);
+
   try {
-    const result = await token.symbol();
-    return result[0];
+    const code = await contract.provider.getCode(tokenAddress);
```

Comment:
The `@metamask/ethjs` library is more lenient and permits interactions with non-existent contracts. This is why token.symbol() previously worked and returned an empty string (""). Now that we’ve removed ethjs, we need to first verify if the contract exists before attempting such calls.